internal/github/pr-helpers.go

```diff
@@ -0,0 +1,45 @@
+package github
```

Comment:
In order to have the functions `GetLatestCommitHash`, `FillEmptyPRField`, `FindOriginalPR` and the tests back in `changelog/builder.go` we would need to skip subfolders in `testdata` since running tests with github creates an additional folder with .yaml files (testdata/fixtures) that can't be parsed.

Reply:
> Is it alright if I modify the walker function to skip the folder?

yes! Please create a separate PR for this but this is a great improvement! I wrote the walker function to be really simple but this ignored a lot of possible edge cases that we need to fix to make the tool resilient.

It may also be meaningful to move testdata changelog fragments to a dedicated folder, like `testdata/fragments`.